apps/modernization-ui/src/design-system/table/preferences/column-preference-panel.module.scss

```diff
@@ -4,6 +4,7 @@
 .panel {
     display: flex;
     flex-direction: column;
+    min-width: 15.5rem;
```

Comment:
The min-width for this panel is currently coming from [here](https://github.com/CDCgov/NEDSS-Modernization/blob/9fb7f87a88dd212763506763b085e95273057ad3/apps/modernization-ui/src/apps/search/layout/result/header/options/table/search-results-table-options.module.scss#L4).  With the reasoning being that the Column preference panel could be any size but for these table options it's in an overlay panel of a certain size.

Reply:
Okay makes sense